packages/ng/package.json

```diff
@@ -18,7 +18,7 @@
   },
   "scripts": {
     "ng": "ng",
-    "start": "gulp generatedoc && ng serve -a=demo",
+    "start": "gulp generatedoc && ng serve -a=demo --port 4242",
```

Comment:
why ?

Reply:
This was commited by mistake. I work on another project using :4200. Sorry about that ☘️